core/src/main/java/io/kestra/core/models/tasks/runners/types/ProcessTaskRunner.java

```diff
@@ -19,17 +20,65 @@
 import java.util.List;
 import java.util.Map;
 
+
+
 @Introspected
 @SuperBuilder
 @ToString
 @EqualsAndHashCode
 @Getter
 @NoArgsConstructor
-// all task runners are beta for now, but this one is stable as it was the one used before
-@Plugin(beta = true, examples = {})
 @Schema(
-    title = "A task runner that runs task as a process on the Kestra host",
-    description = "When the Kestra Worker that runs this process is terminated, the process will be terminated and the task fail."
+    title = "A task runner that runs a task as a sub-process of the Kestra process on the Kestra host.",
```

Comment:
```suggestion
    title = "Task runner that deploys a task as a subprocess on the Kestra host.",
```

Reply:
I don't think deploy is the good verb here.
I understand that a runner that runs is not good, so maybe execute instead of run?

Reply:
sure, execute is good too

just wanted to avoid "a runner running in X runs task" :)